roles/pulp_devel/tasks/install_docs_requirements.yml

```diff
@@ -39,7 +39,7 @@
 
 - name: Install requirements for Azure/S3 tests
   pip:
-    name: django-storages[boto3, azure]
+    name: django-storages[boto3, azure]>=1.12.2
```

Comment:
It would be nice to pull in boto3 when using s3 storage

Reply:
it pulls boto3 when you declare:
```yaml
pulp_install_object_storage: s3
```
in your playbook
https://github.com/pulp/pulp_installer/blob/master/playbooks/example-use/playbook.yml#L24
https://github.com/pulp/pulp_installer/blob/master/roles/pulp_common/tasks/install_pip.yml#L212

Reply:
Nice, thanks. Is that documented?

Reply:
it is poorly documented here:
https://docs.pulpproject.org/pulp_installer/roles/pulp_common/

I'll be adding it to this page:
https://docs.pulpproject.org/pulp_installer/objectstorage/

Reply:
Excellent, thanks